src/components/transportation-modes/TransportModes.tsx

```diff
@@ -40,28 +42,42 @@ export const getTransportModeText = (
   modes: TransportModePair[],
   t: TranslateFunction,
   modesDisplayLimit: number = 2,
+  unknownModeText?: string,
 ): string => {
   const modesCount: number = modes.length;
 
   if (!modes) return '';
   if (modesCount > modesDisplayLimit) {
     return t(FareContractTexts.transportModes.multipleTravelModes);
   }
-  return modes
-    .map((tm) => t(FareContractTexts.transportMode(tm.mode, tm.subMode)))
-    .filter(removeDuplicateStringsFilter)
-    .join('/');
+  if (unknownModeText && modes.map((m) => m.mode).includes('unknown')) {
+    return unknownModeText;
+  }
+  return _.capitalize(
+    modes
+      .map((tm) => t(FareContractTexts.transportMode(tm.mode, tm.subMode)))
+      .filter(removeDuplicateStringsFilter)
+      .join('/'),
+  );
 };
 
 export const TransportModes = ({
   modes,
   iconSize,
   disabled,
+  textType,
+  textColor,
+  unknownModeText,
+  useUnknownIcon = true,
```

Comment:
Could it be an alternative to just not set any `transportModes` on the group that should be "Andre billettyper", and have this handling outside of `TransportModes` component? 

Then it would be something like this:
```ts
fareProductGroup.transportModes ? <TransportModes ...> : <ThemeText>Andre billettyper<ThemeText/>
```
in the `FareProductGroup` component.

It would reduce the complexity a bit in the shared `TransportModes` component, and we probably wouldn't need this `unknownModeText` and `useUnknownIcon` params.